bedrock/firefox/templates/firefox/more.html

```diff
@@ -16,18 +16,51 @@
 {% block content %}
 <main>
   {% call hero(
-    title=ftl('firefox-products-are'),
+    title=ftl('learn-more-about-firefox'),
```

Comment:
You can keep the old string as a fallback for l10n if you want. Whether or not that makes sense depends on how much work has been done to localize the page.